tracer/test/Datadog.Trace.TestHelpers/SpanMetadataRules.cs

```diff
@@ -369,5 +369,13 @@ public static Result IsWebRequest(this MockSpan span) => Result.FromSpan(span)
                 .IsPresent("http.url")
                 .MatchesOneOf("component", "HttpMessageHandler", "WebRequest")
                 .Matches("span.kind", "client"));
+
+        public static Result IsProcessStart(this MockSpan span) => Result.FromSpan(span)
```

Comment:
Is this used anywhere?

Reply:
The integration test code should call this method on all the spans it gets, but now that you mention it, I don't think it's being invoked yet

Reply:
It was used before, but not now that the process integration tests are based on snapshots. This code is not needed anymore. Thanks for spoting it.